index.js

```diff
@@ -22,7 +28,14 @@ function handler(event) {
     var now = Math.round(Date.now() / 1000);
     if (now > parseInt(querystring.exp.value, 10)) {
       parts.splice(-2, 1); // digest is always 2nd to last
-      var headers = { location: { value: `${EXPIRED_REDIRECT_PREFIX}/${parts.join('/')}` } };
+      var value = `${EXPIRED_REDIRECT_PREFIX}/${parts.join('/')}`;
+
+      // preserve token auth, for private feed enclosures
+      if (querystring.auth && querystring.auth.value) {
+        value += `?auth=${querystring.auth.value}`;
```

Comment:
As a bonus - I realized private feed urls that expired would be redirected back to Dovetail Router _without_ any `?auth` param.  Which would get you a 401.

I'm not sure if this is a bad idea or not ... if this CDN url with `?auth=1234&exp=<epoch>` got out into the wild, you'd have a permalink to get a private (possibly adfree) episode.  Is that better than just 401-ing listeners for this edge case?